stand/defs.mk

```diff
@@ -128,6 +128,12 @@ SSP_CFLAGS=
 CFLAGS+=	-ffreestanding ${CFLAGS_NO_SIMD}
 .if ${MACHINE_CPUARCH} == "aarch64"
 CFLAGS+=	-mgeneral-regs-only -ffixed-x18 -fPIC
+# XXXMorello:
+# Disable Morello support in the bootloader. EFI needs to understand
+# capabilities before we can enable this as it may have interrupts enabled
+# so will need to save/restore the full capability registers
```

Comment:
```suggestion
# XXX-Morello: Disable Morello support in the bootloader. EFI needs to
# understand capabilities before we can enable this as it may have interrupts
# enabled so will need to save/restore the full capability registers
```